nixos/modules/module-list.nix

```diff
@@ -1212,6 +1212,7 @@
   ./services/web-apps/zabbix.nix
   ./services/web-servers/agate.nix
   ./services/web-servers/apache-httpd/default.nix
+  ./services/web-servers/authelia.nix
```

Comment:
Authelia is certainly not a webserver.

Reply:
It responds to http requests, it also has a web ui.

I can move it under security if you want, but `web-apps` (like keycloak) sounds bad

Reply:
Yeah, security is probably a better spot. For the package I would think of `pkgs/servers/security/authelia` as a good place.